tests/e2e/delete-identity.test.ts

```diff
@@ -69,8 +69,6 @@ describe("Delete Identity", () => {
       expect(await text("Hide").exists()).toBeTruthy();
 
       expect(!(await link("_delete_").exists())).toBeTruthy();
-
-      done();
```

Comment:
We should also remove it from this part (You forgot some):
```ts
test("It should navigate to the show of the Identity, and delete it", async (done 👈👈👈) => {
```

Reply:
It was done in this commit 9987601